src/test/resources/net/sf/jabref/importer/fileformat/InspecSilverPlatterImporterTest.txt

```diff
@@ -1,30 +0,0 @@
-Record INSPEC
```

Comment:
Isn't that file really not used at the InspecImporter or SilverPlatterImportTests?